src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
```

Comment:
Using wildcard imports like '*' can lead to naming conflicts and make the code less readable. It's recommended to import specific classes.